glide.yaml

```diff
@@ -65,4 +65,19 @@ import:
   version: 0fe204460da2c8fa1babcaac196e694de8f1aaa1
 
 - package: github.com/miekg/dns
-  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
\ No newline at end of file
+  version: db96a2b759cdef4f11a34506a42eb8d1290c598e
+
+- package: github.com/shirou/gopsutil
+  version: 2.1.0
+  subpackages:
+  - load
+
+- package: github.com/davecgh/go-spew
+  version: 2df174808ee097f90d259e432cc04442cf60be21
+  subpackages:
+  - spew
+
+- package: golang.org/x/sys
```

Comment:
Good question. One answer is if one needs to clear open source for use (which usually includes dependents), having specific versions may make that task easier. Non-specification of versions, means versions may get updated more frequently which requires clearance tasks needing to be done more frequently. I think that since we collapse all the dependent vendor directories into one, we can end up with something different than what testify includes. One comment from a glide issue on flattening the tree:

First dependency to declare that it needs a Ref wins
Any subsequent dependency that declares the same package, but a different Ref, will generate a warning
If you want to explicitly control, you should declare the package in the top glide.yaml